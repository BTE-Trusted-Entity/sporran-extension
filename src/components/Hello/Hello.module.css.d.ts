declare namespace HelloModuleCssNamespace {
  export interface IHelloModuleCss {
    text: string;
  }
}

declare const HelloModuleCssModule: HelloModuleCssNamespace.IHelloModuleCss & {
  /** WARNING: Only available when `css-loader` is used without `style-loader` or `mini-css-extract-plugin` */
  locals: HelloModuleCssNamespace.IHelloModuleCss;
};

export = HelloModuleCssModule;
