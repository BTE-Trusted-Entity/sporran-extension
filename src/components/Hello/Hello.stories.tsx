import { storiesOf } from '@storybook/react';
import { Hello } from './Hello';

storiesOf('Hello', module).add('renders', () => {
  return <Hello />;
});
