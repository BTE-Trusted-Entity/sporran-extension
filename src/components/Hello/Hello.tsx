import { FC } from 'react';
import styles from './Hello.module.css';
import { browser } from 'webextension-polyfill-ts';

export const Hello: FC = () => {
  return (
    <div className="row">
      <div className="col-lg-12 text-center">
        <p className={styles.text}>
          Example Extension «{browser.i18n.getMessage('manifest_name')}»
        </p>
      </div>
    </div>
  );
};
