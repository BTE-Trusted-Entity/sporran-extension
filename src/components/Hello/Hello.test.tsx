import { Hello } from './Hello';
import { create } from 'react-test-renderer';

it('Hello renders', () => {
  const tree = create(<Hello />).toJSON();
  expect(tree).toMatchSnapshot();
});
