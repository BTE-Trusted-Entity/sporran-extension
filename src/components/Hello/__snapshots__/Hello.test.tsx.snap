// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hello renders 1`] = `
<div
  className="row"
>
  <div
    className="col-lg-12 text-center"
  >
    <p
      className="text"
    >
      Example Extension «
      Sporran
      »
    </p>
  </div>
</div>
`;
